common/nymsphinx/params/src/packet_sizes.rs

```diff
@@ -89,7 +89,7 @@ pub enum PacketSize {
 impl PartialOrd for PacketSize {
     fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
         // order them by actual packet size
-        self.size().partial_cmp(&other.size())
+        Some(self.cmp(other))
```

Comment:
Are we sure this is doing the same thing as before? We seem to no longer be comparing on `size()`

Reply:
Ah. `Ord` already does it.